pandas/core/indexes/base.py

```diff
@@ -2529,15 +2529,24 @@ def set_value(self, arr, key, value):
     def _get_level_values(self, level):
         """
         Return an Index of values for requested level, equal to the length
-        of the index
+        of the index.
 
         Parameters
         ----------
-        level : int
+        level : int or str
+            ``level`` is either the integer position of the level in the 
+            MultiIndex, or the name of the level.
 
         Returns
         -------
         values : Index
+            Because there is only one level when the index has one level,
+            the return value is always self in this case. 
```

Comment:
I would simplify to just:

"self, as there is only one level in the Index"

Reply:
Yes, thats better